templates/vercel-postgres/src/payload.config.ts

```diff
@@ -26,7 +26,7 @@ export default buildConfig({
   plugins: [
     vercelBlobStorage({
       collections: {
-        [Media.slug]: true,
+        media: true,
```

Comment:
Does TS still allow using `Media.slug` without error?

Reply:
No, because `Media.slug` itself has a type of `string`, not `'media'`. Other way around it is to write collection like this:
<img width="632" alt="image" src="https://github.com/user-attachments/assets/34921ec4-e67a-40ac-b4a8-5de28d1607ae">

It would work if the `slug` property itself of `CollectionConfig` would be of the `CollectionSlug` type, though.
But the drawback of this is that you wouldn't be able to define a new collection slug without typescript error, which can be fixed only after type generation.